charts/newrelic-infrastructure/flex/nri-pixie-check.yaml

```diff
@@ -0,0 +1,27 @@
+---
+# This Flex config performs periodic checks of the Pixie
+# healthz and statusz endpoints.  A status for each endpoint
+# is sent to New Relic in a pixieHealthCheck event.
+#
+# If Pixie is not installed in the cluster, no events will be generated.
+discovery:
+  command:
+    exec: /var/db/newrelic-infra/nri-discovery-kubernetes
+    match:
+      label.name: vizier-cloud-connector
+integrations:
+  - name: nri-flex
+    interval: 60s
+    config:
+      name: pixie-health-check
+      apis:
+        - event_type: pixieHealth
+          commands:
+            - run: curl --insecure -s https://${discovery.ip}:50800/healthz | xargs | awk '{print "cloud_connector_health:"$1}'
+              split_by: ":"
+          merge: pixieHealthCheck
+        - event_type: pixieStatus
+          commands:
+            - run: curl --insecure -s https://${discovery.ip}:50800/statusz | awk '{if($1 == ""){ print "cloud_connector_status:OK" } else { print "cloud_connector_status:"$1 }}'
```

Comment:
I remember some weirdness could happen with unscaped `$` symbols (like in `$1` here) in pixie/agent configs, but unfortunately not the details of it. Feel free to ignore if you have tested this to be working.